documentation/analytics/recipes/user-retention.md

```diff
@@ -0,0 +1,118 @@
+# User Retention
+
+This recipe provides information on one of the most important metrics for a successful free-to-play game.
+The user retention gives an indication on the level of engagement and the proportion of users returning to your game.
+
+## End Result
+
+You will answer the following question: The percentage of users who showed up again X days after installing/playing the game. X is typically 1, 7, 30 and 90. There are multiple ways to calculate the user retention ([User Retention: Yes, But Which One?](http://www.applift.com/blog/user-retention)); we will make use of the **classic retention** to calculate the user retention: Which proportion of users come back to the game on Day+X. For instance, for day 7 retention (i.e. X=7) we are interested in the proportion of people who come back precisely on day 7. It does not matter if they have played the game again on day 3 or any other day inbetween and how often before day 7:
+
+![Classic Retention with X=28](../../images/analytics/retention/ClassicRetention.png)
```

Comment:
I think it's good that we reference other's work on the subject, but I wouldn't feel comfortable approving a PR that contains others images. Perhaps we can just re-draw them ourselves. Can I help you with that?